frontend/tiggle/src/store/detailPage.ts

```
@@ -1,26 +1,13 @@
 import { createSlice } from "@reduxjs/toolkit";
 
-import { TxType } from "@/types";
+export interface DetailPageState {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)